src/main/perf/LineFileDocs.java

```diff
@@ -641,6 +685,16 @@ public Document nextDoc(DocState doc) throws IOException {
         }
       }
 
+      if (facetFields.containsKey("RandomLabel")) {
+        int flag = facetFields.get("RandomLabel");
+        if ((flag & 1) != 0) {
+          doc2.add(new FacetField("RandomLabel.taxonomy", randomLabel));
+        }
+        if ((flag & 2) != 0) {
+          doc2.add(new SortedSetDocValuesFacetField("RandomLabel.sortedset", randomLabel));
```

Comment:
Also, I think we are now indexing this label up to four times on each doc?  Stored field, SDV, SSDV, taxo facets?  Hopefully we can get that down to always two (taxo facets, SSDV).